distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -174,15 +181,16 @@ attempt_startup() {
 attempt_shutdown() {
     if is_restarting; then
         echo "Restarting JVM..."
-    else
         if is_managing_solr; then
             stop_solr
```

Comment:
move out of if block

Reply:
That should be fine. Solr has a quick startup time, so starting and stopping it shouldn't cause any pain.